src/compile/axis.ts

```diff
@@ -186,6 +186,14 @@ export function tickSize(model: Model, channel: Channel) {
   return undefined;
 }
 
+export function tickSizeEnd(model: Model, channel: Channel) {
```

Comment:
I mean move `'tickSizeEnd'` from line 75 to line 73 and keep them in alphabetical order.  No need to move the whole method.  :)